doc/data/messages/c/consider-using-generator/details.rst

```diff
@@ -0,0 +1,5 @@
+Removing ``[]`` inside calls that can use containers or generator indistinctly should be considered for performance reasons since a generator will have an upfront cost to pay. The performance will be better if
```

Comment:
```suggestion
Removing ``[]`` inside calls that can use containers or generators should be considered for performance reasons since a generator will have an upfront cost to pay. The performance will be better if
```